src/Sylius/Bundle/CoreBundle/Migrations/Version20240918120744.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240918120744 extends AbstractMigration
```

Comment:
I checked it on both versions earlier and it is actually the same as for MySQL, thats why i used `Doctrine\Migrations\AbstractMigration` rather than `Sylius\Bundle\CoreBundle\Doctrine\Migrations\AbstractMigration`, but separated it anyways as I don't remember the approach in that case as we used to do earlier 😅